src/System.Diagnostics.Process/src/System/Diagnostics/ThreadInfo.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Diagnostics
     /// <internalonly/>
     internal sealed class ThreadInfo
     {
-        internal int _threadId;
+        internal ulong _threadId;
```

Comment:
I see, so where I wrote ```I'd understand it if storing it as a ulong meant we could then use that ID elsewhere more appropriately on OSX``` that is in fact why this change is being made, because we're already using that ID value and it's currently failing without this change?  If so, makes sense.